benchmarks/free_surface_tractions/viscous/free_surface_viscous_cylinder_2D_loading.prm

```diff
@@ -0,0 +1,159 @@
+# This model applies a surface load (using the traction
+# boundary condition) on a free surface overlying 
+# a viscous box. It approximates the analytical solution
+# of Haskell (1935a) for an instantaneously applied
+# axisymmetric cylindrical load over a viscous half-space.
+
+set Dimension                              = 2
+set Start time                             = 0
+set End time                               = 1500
+set Use years in output instead of seconds = true
+set Nonlinear solver scheme                = single Advection, single Stokes
+set CFL number                             = 0.5
+set Maximum time step                      = 10
+set Output directory                       = output_free_surface_viscous_cylinder_2D_loading
+set Timing output frequency                = 1
+set Pressure normalization                 = no
+
+# Model geometry (500x500 km, 50 km spacing)
+subsection Geometry model
+  set Model name = box
+
+  subsection Box
+    set X repetitions = 10
+    set Y repetitions = 10
+    set X extent      = 500e3
+    set Y extent      = 500e3
+  end
+end
+
+# Mesh refinement specifications 
+subsection Mesh refinement
+  set Initial adaptive refinement        = 1
```

Comment:
Do the results change significantly if you do not do the adaptive refinement? If so, it might be worth putting in a brief comment mentioning this.